posts/2017-09-11-serverless-api-gateway-domain.md

```diff
@@ -0,0 +1,210 @@
+---
+title: How to set up a custom domain name for Lambda and API Gateway with Serverless
+description: Use the Serverless Framework to configure a clean domain name on API Gateway.
```

Comment:
"Learn how to set up a custom domain name for AWS Lambda & API Gateway using the Serverless Framework to configure a clean domain name for your services" (something like this)